core-application/src/main/java/gohigher/application/port/in/CalenderApplicationRequest.java

```diff
@@ -0,0 +1,34 @@
+package gohigher.application.port.in;
+
+import static gohigher.application.ApplicationErrorCode.*;
+
+import gohigher.global.exception.GoHigherException;
+import lombok.Getter;
+
+@Getter
+public class CalenderApplicationRequest {
+
+	private final Long userId;
+	private final int year;
+	private final int month;
+
+	public CalenderApplicationRequest(Long userId, int year, int month) {
+		validateNegativeYear(year);
+		verifyMonthIsInValidRange(month);
+		this.userId = userId;
+		this.year = year;
+		this.month = month;
+	}
+
+	private void validateNegativeYear(int year) {
+		if (year <= 0) {
+			throw new GoHigherException(INVALID_DATE_INFO);
+		}
+	}
+
+	private void verifyMonthIsInValidRange(int month) {
+		if (month < 1 || month > 12) {
+			throw new GoHigherException(INVALID_DATE_INFO);
+		}
+	}
```

Comment:
```suggestion
	private void validateNegativeYear(int year) {
		if (year <= 0) {
			throw new GoHigherException(INVALID_DATE_INFO);
		}
	}

	private void validateMonthIsWithinRange(int month) {
		if (month < 1 || month > 12) {
			throw new GoHigherException(INVALID_DATE_INFO);
		}
	}
```

해당 검증 로직은 DTO 가 아닌 도메인이 어울릴듯 합니다! 그리고 검증 테스트 관련해서는 용어(validate) 를 통일하면 어떨까요!

Reply:
해당 메서드도 validate~로 수정했습니다!!